src/rgw/setup/rgw.py

```diff
@@ -620,6 +629,12 @@ def _logrotate_generic(conf: MappedConf):
             Log.info(f'{const.LOGROTATE_TMPL} file copied to {const.LOGROTATE_CONF}')
         except Exception as e:
             Log.error(f"Failed to configure logrotate for {const.COMPONENT_NAME}. ERROR:{e}")
+        # start cron.d service
+        try:
+            os.system(f"chmod +x {const.CRON_LOGROTATE}")
```

Comment:
os.system() needs to be replaced with SimpleProcess calls @sachitanands anywhere in code, historically many security concerns were raised with it. Plan to remove it in future PR,